.github/workflows/code-formatting.yml

```diff
@@ -9,14 +9,14 @@ jobs:
     timeout-minutes: 15
     steps:
     - name: Checkout
-      uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4
+      uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
       with:
         ref: ${{ github.ref }}
 
     - name: Install .NET Core
-      uses: actions/setup-dotnet@3447fd6a9f9e57506b15f895c5b76d3b197dc7c2 # v3
+      uses: actions/setup-dotnet@3951f0dfe7a07e2313ec93c75700083e2005cbab # v4
       with:
-        dotnet-version: 6.x
+        dotnet-version: 8.x
```

Comment:
I'm not sure for this one. I think this is why the pipelines are failing

Reply:
This is only for the formatting job. Not the tests.
By the way, I don't understand why they fail as Nunit tells that all tests pass...

Reply:
Not sure, but dorny/test-reporter should be upgraded to v2 as well I think

Reply:
I've just took the 2 commits from renovate for the actions